packages/polar/src/lib/client.ts

```diff
@@ -14,8 +14,8 @@ export async function getSigningClient (
   account: Account
 ): Promise<SecretNetworkClient> {
```

Comment:
Has secretjs changed the object creation again ?
If this is the case then we don't need Promise in functions. Please update this

Reply:
okay